core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -957,6 +957,17 @@ func (o *evmTxStore) FindReceiptsPendingConfirmation(ctx context.Context, blockN
 	return
 }
 
+func (o *evmTxStore) FindHighestSequence(ctx context.Context, fromAddress common.Address, chainId *big.Int) (nonce evmtypes.Nonce, err error) {
+	var cancel context.CancelFunc
+	ctx, cancel = o.mergeContexts(ctx)
+	defer cancel()
+	qq := o.q.WithOpts(pg.WithParentCtx(ctx))
+
+	sql := `SELECT MAX(nonce) FROM evm.txes WHERE from_address = $1 and evm_chain_id = $2`
+	err = qq.Get(&nonce, sql, fromAddress.String(), chainId.String())
```

Comment:
So lets say there was no entry of evm.txes found for an address. It can happen if the address hasn't been used for some time, and the Reaper cleaned up previous txes that were finalized.
In that case, this query will return 0.
Thus, the caller of this function should handle the 0 case, by fetching nonce from onchain, right?